src/building/house_service.c

```diff
@@ -156,6 +157,44 @@ void house_service_calculate_culture_aggregates(void)
                 }
             }
         }
+        
+        if (b->house_population > 0) {
+			if (b->data.house.theater) {
+				if (b->subtype.house_level >= HOUSE_LARGE_VILLA) {
+					b->house_theater_accessibility = 96;
```

Comment:
Maybe you should add a define both to 96 and to 168? They are magic numbers, but what do they represent?